nexus/pkg/artifacts/manifest.go

```diff
@@ -80,3 +81,14 @@ func (m *Manifest) WriteToFile() (filename string, digest string, rerr error) {
 	digest, rerr = utils.ComputeB64MD5(data)
 	return
 }
+
+func (m *Manifest) GetManifestEntryFromArtifactFilePath(path string) (ManifestEntry, error) {
+	manifestEntries := m.Contents
+	manifestEntry, ok := manifestEntries[path]
+	if !ok {
+		// implement _get_obj_entry
```

Comment:
`WBValue` is a layer on top of artifacts which maps `WBValue` instances to files stored in artifacts. We're dealing with files here, we don't have to worry about this.